lib/services.rb

```diff
@@ -4,7 +4,11 @@ module Services
   def self.publishing_api
     @publishing_api ||= GdsApi::PublishingApiV2.new(
       Plek.new.find('publishing-api'),
-      disable_cache: true,
+      cache_ttl: 5.minutes,
```

Comment:
Good point - do you know how publishing apps handle this? I don't see a mechanism for caching just certain calls in gds-api-adapters, but will look into it.